op-challenger/runner/metrics.go

```diff
@@ -43,6 +44,11 @@ func NewMetrics(runConfigs []RunConfig) *Metrics {
 		ContractMetrics: contractMetrics.MakeContractMetrics(Namespace, factory),
 		VmMetrics:       metrics.NewVmMetrics(Namespace, factory),
 
+		up: factory.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Name:      "up",
+			Help:      "The VM runner has started to run",
+		}, []string{"vm"}),
```

Comment:
I wouldn't parameterise this.  Either the whole box is up or the box is down - it can't be up for one vm and not for another.